docs/hpc/systems/supported/catscan.md

```diff
@@ -0,0 +1,113 @@
+# Catscan
+
+The Catscan cluster is an XCAT stand alone cluster.
+
+**Login node**: catscan.lbl.gov
+
+Sports a 269Tb zfs filesystem for data and computational scratch space on /pool0 (see below)
+
+Compute nodes: n0000, n0001, n0002, n0003
+
+## Cluster Configuration:
```

Comment:
In terms of formatting style, I have been omitting the colon at the end of the headings.
```suggestion
## Cluster Configuration
```